lib/hotwire/spark/installer.rb

```diff
@@ -6,21 +6,25 @@ def initialize(application)
   end
 
   def install
+    configure_cable_server
     configure_middleware
     monitor_paths
   end
 
   def configure_middleware
```

Comment:
This method can be private (we used to invoke this from the tests but we don't do it anymore). Handled in https://github.com/basecamp/hotwire-spark/commit/0aed259daca8e2e8d328d9334e52d713513af7c7